pkg/util/log/logcrash/crash_reporting.go

```diff
@@ -263,6 +264,17 @@ func SetupCrashReporter(ctx context.Context, cmd string) {
 			"buildchannel": info.Channel,
 			"envchannel":   info.EnvChannel,
 		})
+
+		rawTags := envutil.EnvOrDefaultString("COCKROACH_CRASH_REPORT_TAGS", "")
+		if len(rawTags) > 0 {
```

Comment:
nit: extract the parsing logic into a function so you can test it. That allows you to simplify the untested logic to something like:

```
scope.SetTags(getTagsFromEnvironment())
```

Reply:
Done!